core/settings.py

```diff
@@ -1,9 +1,16 @@
 # Django settings
```

Comment:
Could we put the settings file somewhere else instead of putting it inside a **Django App**?

Reply:
@saadmk11 are you thinking at the top level with `manage.py` or something else?

Reply:
> @saadmk11 are you thinking at the top level with `manage.py` or something else?

I think at the top level is fine. I tend to put these in their own app (I prefer `config` over `djangopackage` but in this case the `settings/__init__.py` is roughly the same as `settings.py` from a namespace perspective. 

What I care the most about is that we are consistent and update everywhere this needs to be reflected in.